install_config/install/prerequisites.adoc

```diff
@@ -783,6 +789,9 @@ This will destroy any Docker containers or images currently on the host.
 # rm -rf /var/lib/docker/*
 # systemctl restart docker
 ----
++
+If there is any content in *_/var/lib/docker/*_*, it has to be deleted. Files
```

Comment:
Is that an errant asterisk after the file path? Should it be *_/var/lib/docker/_* ?